pikaur/print_department.py

```diff
@@ -602,7 +602,7 @@ def get_aur_sort_key(pkg: AURPackageInfo) -> Tuple[AurSortKey, str]:
             ) and (
                 package.popularity is not None
             ):
-                rating = color_line('({}, {:.2f})'.format(
+                rating = color_line('({}, {:.2f}) '.format(
```

Comment:
why to add the space to the of `rating`?

Reply:
So that if `LastUpdated` is displayed, it would happen after a space in `rating`. I'm following the same convention as `installed` above, where the space is added on the variable before rather than after

Reply:
since the field is optional, it might be better to prepend space to last_updated string

also since it's a separate option now, you can return to previous idea, printing it on the new line (ie prepend \n except for space)